src/scrapers/platformen/scraper.py

```diff
@@ -212,6 +212,14 @@ def add_source(self, source: PlatformSource):
             raise ValueError("source is None")
         self._sources.append(source)
 
+    def remove_sources(self, source: PlatformSource):
+        if source is None:
+            raise ValueError("source is None")
+        self._sources.remove(source)
```

Comment:
Let op dat `remove` maar 1x iets weghaalt uit een lijst, dus zou je moeten checken bij het inserten dat er geen duplicaat in komt

Reply:
Je zou anders ook een set kunnen gebruiken in plaats van een list

Reply:
Heb extra check op de add_source gezet.